shared/utils/yaml-to-shorthand.py

```diff
@@ -130,8 +129,8 @@ def to_xml_element(self):
         title.text = self.title
         add_sub_element(value, 'description', self.description)
 
-        if self.id_ is not "conditional_clause":
-            value = add_warning(value, self.warning)
+        if self.id_ != "conditional_clause":
+            add_warnings(value, self.warnings)
```

Comment:
why is this condition even here? will it pick up any warnings for conditional clause?

Reply:
@mpreisler yes.... conditional_clause kept showing up and throwing errors.